### PR TITLE
[meta]: Add support of SAI_ATTR_VALUE_TYPE_ACL_RESOURCE_LIST attribute

### PR DESCRIPTION
**Pull request by volodymyrsamotiy**
Signed-off-by: Volodymyr Samotiy <volodymyrs@mellanox.com>

### CODE REVIEW

**Comment by lguohan** [comment]
@volodymyrsamotiy , there are conflict due to my rebase, can you resolve the conflict?

**Comment by volodymyrsamotiy** [comment]
@lguohan, conflicts are already resolved.

File: meta/saiserialize.cpp [hunk 1]
```diff
@@ -1217,6 +1264,9 @@ std::string sai_serialize_attr_value(
         case SAI_ATTR_VALUE_TYPE_QOS_MAP_LIST:
             return sai_serialize_qos_map_list(attr.value.qosmap, countOnly);
 
+        case SAI_ATTR_VALUE_TYPE_ACL_RESOURCE_LIST:
+           return sai_serialize_acl_resource_list(attr.value.aclresource, countOnly);
```
Comment (lguohan) [inline meta/saiserialize.cpp]: >return sai_serialize_acl_resource_list(attr.value.aclresource, countOnly); [](start = 11, length = 74)

spacing does not seem right?

File: meta/saiserialize.cpp [hunk 2]
```diff
@@ -2123,6 +2242,9 @@ void sai_deserialize_attr_value(
         case SAI_ATTR_VALUE_TYPE_QOS_MAP_LIST:
             return sai_deserialize_qos_map_list(s, attr.value.qosmap, countOnly);
 
+        case SAI_ATTR_VALUE_TYPE_ACL_RESOURCE_LIST:
+           return sai_deserialize_acl_resource_list(s, attr.value.aclresource, countOnly);
```
Comment (lguohan) [inline meta/saiserialize.cpp]: >  return sai_deserialize_acl_resource_list(s, attr.value.aclresource, countOnly); [](start = 9, length = 81)

spacing is not correct

**Comment by lguohan** [comment]
@volodymyrsamotiy , can you correct the spacing issues

**Comment by volodymyrsamotiy** [comment]
@lguohan, spacing issues are already corrected

**Pull request merged**